crates/yakui-vulkan/src/lib.rs

```diff
@@ -364,8 +305,21 @@ impl YakuiVulkan {
             .depth_stencil_state(&depth_state_info)
             .color_blend_state(&color_blend_state)
             .dynamic_state(&dynamic_state_info)
-            .layout(pipeline_layout)
-            .render_pass(render_pass);
+            .layout(pipeline_layout);
+        let rendering_info_formats;
+        let mut rendering_info;
+        if let Some(format) = options.dynamic_rendering_format {
+            rendering_info_formats = [format];
+            rendering_info = vk::PipelineRenderingCreateInfo::builder()
+                .color_attachment_formats(&rendering_info_formats);
+            graphic_pipeline_info = graphic_pipeline_info.push_next(&mut rendering_info);
+        } else {
+            debug_assert!(
```

Comment:
Is there any reason why we wouldn't make this a regular assert?

Since this is only going to be called once (ish), it seems like it would make more sense to just check this rather than silently accepting a configuration that could never work.

Reply:
Good call; fixed.